src/style.css

```
@@ -1 +1,70 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 50px;
+  margin-right: 50px;
```

This should help with first two tests.
```suggestion
  padding: 0 50px;
```